kubernetes/tools/devaccount/deploy-dev-account-payload.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+kubectl apply -k .
+
+mkdir -p /home/waggle/.kube
```

Comment:
This implies that the `waggle` user already exists, and that basically the node is running 1.8.2 OS.

How do you want to handle this (if at all) on nodes that don't have this update yet?

Reply:
I think you should only proceed on nodes that already have the `waggle` user.

Reply:
Yes, I added a user check. Thanks!